docs/tools/codetools/README.md

```diff
@@ -3,12 +3,45 @@
 **Install Flake8**:
 https://flake8.pycqa.org/en/latest/
 
-```
+```bash
 python -m pip install flake8
 ```
 
 **Use Flake8 to check your code before submission**:
 
-```
+```bash
 flake8 --config .flake8 [file_name]
 ```
+
+# Use pre-commit hooks for code consistency
+
+**Install pre-commit**:
+https://pre-commit.com/
+
+```bash
+pip install pre-commit
+```
+
+**Install the git hooks script**:
+
+```bash
+pre-commit install
```

Comment:
When testing with mac, the following error appears: "pre-commit not found"
The error can be fixed using: https://github.com/pre-commit/pre-commit/issues/1537